### ISSUE TITLE
No source removal if no modules

### ISSUE THREAD

**Issue by sveyret** [projg2/eclean-kernel#40]
Hi,
If I understand the source code correctly, the kernel source directory (`/usr/src/linux-xxx`) is tied to the module directory (`/lib/modules/xxx`) because it is read from a link (named `build`).
On one of my server, I have a kernel where `CONFIG_MODULES` is disabled. In this case, there is no modules directory and so, `eclean-kernel` is never removing my old sources.
Is there currently a way to say to `eclean-kernel` to remove the source directories without looking for the modules? Or could that be an option?
Thank you.
```bash
> eclean-kernel -l
6.1.38-gentoo [6.1.38-gentoo]
- systemmap: /boot/System.map-6.1.38-gentoo
- vmlinuz: /boot/vmlinuz-6.1.38-gentoo
- last modified: 2023-07-14 13:09:46
6.1.31-gentoo [6.1.31-gentoo]
- systemmap: /boot/System.map-6.1.31-gentoo
- vmlinuz: /boot/vmlinuz-6.1.31-gentoo
- last modified: 2023-06-03 12:47:53
```

**Comment by mgorny** [comment]
No, there's no such option. Source directories are largely unpredictable, so we rely on these symlinks to tell us which one is correct.

**Comment by bstaletic** [comment]
With newer kernels `make module_install` works even with `CONFIG_MODULES` not set.
I do not know who sets up the symlinks, but if one is using `gentoo-kernel` to manage kernels, this issue does not exist since https://github.com/gentoo/gentoo/pull/32358
